items/ITEM_SPIRIT_BOW.json

```diff
@@ -0,0 +1,37 @@
+{
+  "itemid": "minecraft:bow",
+  "displayname": "§f§f§6Spirit Bow",
+  "nbttag": "{overrideMeta:1b,HideFlags:254,display:{Lore:[0:\"§7Gear Score: §d350 §8(508)\",1:\"§7Damage: §c+210 §8(+317.1)\",2:\"§7Strength: §c+100 §8(+151)\",3:\"\",4:\"§7Deals §a+2% §7damage to §aUndead\",5:\"§amonsters §7for every §a1% §7of\",6:\"§7your missing health.\",7:\"\",8:\"§7§bSpirit Item: §7When turned\",9:\"§7into a ghost, this item becomes\",10:\"§7a Ghost Ability.\",11:\"\",12:\"§6Spirit Ability: Spirit Bomb\",13:\"§7Shoots a spirit that deals\",14:\"§7§c8000 §7damage on impact.\",15:\"\",16:\"§7§8This item can be reforged!\",17:\"§aPerfect 70000 / 70000\",18:\"§7§4❣ §cRequires §aCatacombs Floor IV\",19:\"§aCompletion\",20:\"§6§lLEGENDARY DUNGEON BOW\"],Name:\"§f§f§6Spirit Bow\"},ExtraAttributes:{originTag:\"UNKNOWN\",id:\"ITEM_SPIRIT_BOW\",uuid:\"8d488463-a40d-45cb-a422-df5bb38b8ae2\",timestamp:\"8/1/20 3:53 PM\"},AttributeModifiers:[]}",
+  "damage": 0,
+  "lore": [
+    "§7Gear Score: §d350 §8(508)",
+    "§7Damage: §c+210 §8(+317.1)",
+    "§7Strength: §c+100 §8(+151)",
+    "",
+    "§7Deals §a+2% §7damage to §aUndead",
+    "§amonsters §7for every §a1% §7of",
+    "§7your missing health.",
+    "",
+    "§7§bSpirit Item: §7When turned",
+    "§7into a ghost, this item becomes",
+    "§7a Ghost Ability.",
+    "",
+    "§6Spirit Ability: Spirit Bomb",
+    "§7Shoots a spirit that deals",
+    "§7§c8000 §7damage on impact.",
+    "",
+    "§7§8This item can be reforged!",
+    "§aPerfect 70000 / 70000",
+    "§7§4❣ §cRequires §aCatacombs Floor IV",
+    "§aCompletion",
+    "§6§lLEGENDARY DUNGEON BOW"
+  ],
+  "internalname": "ITEM_SPIRIT_BOW",
+  "crafttext": "",
+  "clickcommand": "viewrecipe",
```

Comment:
Recipe doesnt work either thats not the internal name or it doesnt have a recipe.